pkgs/development/php-packages/pdlib/default.nix

```diff
@@ -0,0 +1,24 @@
+{ buildPecl, lib, pkgs }:
+let
+  pname = "pdlib";
+  version = "1.0.2";
+in
+buildPecl {
+  inherit pname version;
```

Comment:
```suggestion

buildPecl {
  pname = "pdlib";
  version = "1.0.2";
```

Reply:
Applying this diff would break eval because version is needed as a variable in the `src` attribute, so it would require to make it a `rec`ursive attribute, which isn't included in this patch.

Also, I'm more in favor of the style that @aanderse used in this PR to begin with to make the scope simpler.